mentat/include_files.py

```diff
@@ -126,13 +126,16 @@ def validate_and_format_path(
     """
     path = Path(path)
 
+    # Resolve ~
+    path = path.expanduser()
```

Comment:
```suggestion
    try:
        path = path.expanduser()
    except RuntimeError as e:
        raise PathValidationError(f"Error expanding path {path}.")
```
I was getting familiar with `expanduser` and see it might raise a RuntimeError. It's a rare case, but we're not handling them.

Reply:
+1